server/config/config.go

```diff
@@ -134,6 +134,8 @@ type LoggingConfig struct {
 	JSON                 bool
 	DisableBanner        bool          `yaml:"disable_banner"`
 	ErrorRetentionPeriod time.Duration `yaml:"error_retention_period"`
+	TracingEnabled       bool          `yaml:"tracing_enabled"`
+	TracingType          string        `yaml:"tracing_type"`
```

Comment:
Even if the other exported fields don't have docs, please document these two (maybe state possible valid values of tracing_type)

Reply:
Where would this be documented?

Reply:
Ah, sorry, I meant the usual go docs of exported fields:
```go
// TracingEnabled ...
TracingEnabled bool ...
// TracingType ...
TracingType string ...
```